arch/platform/cooja/Makefile.cooja

```diff
@@ -72,9 +72,8 @@ ifneq ($(MAKECMDGOALS),clean)
 
```

Comment:
This is a really nice improvement, can the `ifneq ($(MAKECMDGOALS),clean)` also be removed?

Reply:
Yes, that is no longer needed. Fixed now. Thanks for the suggestion!